signer/cosigner_nonce_cache.go

```diff
@@ -88,6 +95,17 @@ func (nc *NonceCache) Set(uuid uuid.UUID, cn *CachedNonce) {
 	nc.cache[uuid] = cn
 }
 
+func (nc *NonceCache) GetSortedByExpiration() []*CachedNonce {
```

Comment:
Should we add an enhancement issue to remove this sorting and change the CosignerNonceCache's NonceCache's cache from a map to an array which would be FIFO and pruning can be optimized?

Reply:
yes, this makes sense. good call